tools/update-requirements.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/env bash
+#
+# Updates the requirements.txt file. Is run from the Makefile and must run on
+# the top-level.
+#
+# Update the script when font production relies on more than fontmake and
+# vttLib.
+
+if [ ! -f requirements.txt ]
+then
+  echo "This script must be run in the same directory that contains the \
+requirements.txt file."
+  exit 1
+fi
+
+TMPDIR=$(mktemp -d)
+python3 -m venv $TMPDIR
```

Comment:
I think just `python` is fine, so that it will work from within a virtual environment -- which is the only environment where it makes sense to actually pip install from a requirements.txt file with concrete versions. You wouldn't want your global or user's environment to change according to the font project (or revision of it) that you're are currently building.